content/en/docs/contribute/localization.md

```diff
@@ -73,7 +73,9 @@ For an example of adding a label, see the PR for adding the [Italian language la
 
 ### Find community
 
-Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). Other localization teams are happy to help you get started and answer any questions you have.
+Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/sig-docs) and the [SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations). Other localization teams are happy to help you get started and answer any questions you have.
+
+Please also consider participating in the [SIG Docs Localization Subgroup meeting](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/).  The mission of the SIG Docs localization subgroup is to work across the SIG Docs localization teams to collaborate on defining and documenting the processes for creating localized contribution guides. In addition, the SIG Docs localization subgroup will look for opportunities for the creation and sharing of common tools across localization teams and also serve to identify new requirements to the SIG Docs Leadership team.
```

Comment:
This one probably when we state please consider join the meeting we can just tell the folks that they can see the details in the slack or by joining google groups so we don’t need to attach the docs directly. I just think about this when @sftim mentioned this

Reply:
@irvifa I could change the link to point to https://github.com/kubernetes/community/tree/master/sig-docs which currently lists all our google docs and all our meeting times.   Would that help?

Reply:
@irvifa The advantage of changlng the link to be https://github.com/kubernetes/community/tree/master/sig-docs is that   if the google doc link eventually changes I only have to update it in one place (on the community site) instead of two places (the community site and the localization page)

Reply:
@irvifa I have removed the google doc link and instead reference the standard  https://github.com/kubernetes/community/tree/master/sig-docs page and also mentioned for folks to ask questions on the slack channel